rules/opentitan/defs.bzl

```diff
@@ -120,7 +120,9 @@ def _parameter_name(env, pname):
 def _hacky_tags(env):
     (_, suffix) = env.split(":")
     tags = []
-    if suffix.startswith("fpga_cw310_") or suffix.startswith("fpga_cw340_"):
```

Comment:
It is possible that master and es have diverged at this point but in master we have a slightly better way of doing hacky_tags (still hacky): https://github.com/lowRISC/opentitan/blob/cf32d0f1190b72144c52ffe1b226b4fe37eb3988/rules/opentitan/defs.bzl#L142

Reply:
They have diverged and from a first look, the function on master looks better.  I don't think I want to dive down the rabbithole of resolving this divergence.